lib/vnet/daemon/protocol_darwin.h

```diff
@@ -0,0 +1,21 @@
+#ifndef TELEPORT_LIB_VNET_DAEMON_PROTOCOL_DARWIN_H_
+#define TELEPORT_LIB_VNET_DAEMON_PROTOCOL_DARWIN_H_
+
+#import <Foundation/Foundation.h>
```

Comment:
Why is foundation required here? Is it because of `@protocol` ?

Reply:
Ah, it's not needed. It must be a leftover from when I was experimenting with different approaches to passing the config to the daemon.